smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/ServiceGenerator.kt

```diff
@@ -68,6 +73,46 @@ class ServiceGenerator(
                 outputShapeName
             )
         }
+
+        /**
+         * Helper method for generating in-line documentation for operation
+         */
+        private fun renderOperationDoc(model: Model, service: ServiceShape, op: OperationShape, writer: SwiftWriter) {
+            writer.writeShapeDocs(op)
+            writer.writeAvailableAttribute(model, op)
+
+            fun writeEmptyLine() {
+                writer.writeSingleLineDocs { write("") }
+            }
+
+            writeEmptyLine()
+            writer.writeDocs("\\- Parameter ${op.inputShape.name} : ${retrieveMemberShapeDoc(op.inputShape, model)}")
+
+            writeEmptyLine()
+            writer.writeDocs("\\- Returns: \\`${op.outputShape.name}\\` : ${retrieveMemberShapeDoc(op.outputShape, model)}")
+
+            if (op.getErrors(service).isNotEmpty()) {
+                writeEmptyLine()
+                writer.writeSingleLineDocs { write("- Throws: `${op.toUpperCamelCase() + "Error"}` : Place-holder wrapper object for possible exceptions listed below.") }
```

Comment:
What is this about a "place-holder wrapper"?  We should be throwing the individual exceptions unwrapped.

Reply:
Got on huddle call to determine desired error output, reflected in PR post.